update Installing the appliance (#35)
anbox-cloud/discourse-docs: tutorial/installing-appliance.md

The Anbox Cloud Appliance provides a deployment of Anbox Cloud to a single machine. This offering is well suited for initial prototype and small scale deployments.

[note type="information" status="Note"]

> There is a difference between the Anbox Cloud Appliance and the full Anbox Cloud installation (see [Variants](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802#variants)). This section focuses on the **Anbox Cloud Appliance**. For instructions on how to install the **Anbox Cloud Appliance**, see [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/install-with-juju/17744).

Reviewer: ```suggestion
There are differences between the Anbox Cloud Appliance and the full Anbox Cloud installation (see [Variants](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802#variants)). This section focuses on the **Anbox Cloud Appliance**. For instructions on how to install the **Anbox Cloud Appliance**, see [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/install-with-juju/17744).
```
